consopt.go

```diff
@@ -97,10 +101,11 @@ func WithSize(size ...int) ConsOpt {
 			l.dims = size[0]
 			return l, nil
 		case sizeSetter:
-			l.SetSize(size[0])
-			return layer, nil
+			return layer, l.SetSize(size[0])
 		case Pass:
 			return layer, nil
+		case *Conv:
```

Comment:
no longer necessary given that `*Conv` is a `sizeSetter`

Reply:
I added it because I need to set 2 sizes not a single values and sizeSetter just uses size[0]